website/docs/docs/cloud/dbt-copilot-faqs.md

```diff
@@ -127,3 +127,32 @@ Clients who signed with terms after January 2024 don't need additional terms pri
 Future releases are planned that may bring <Constant name="copilot" /> to even more parts of the <Constant name="cloud" /> application.
 
 </Expandable>
+
+## Copilot Whitelisting
+
+<Expandable alt_header="Whitelisting URLs">
```

Comment:
I think we officially use "allowlisting" in the product, but I can't see the IP restrictions page to verify
```suggestion
<Expandable alt_header="Allowlisting URLs">
```